src/common/style/vanilla/forms.scss

```diff
@@ -0,0 +1,107 @@
+@import 'vanilla-framework/scss/settings';
+@import 'vanilla-framework/scss/base_forms.scss';
+
+@mixin theme-p-forms {
```

Comment:
Hey @bartaz so these will be pulled straight from VF soon. https://github.com/vanilla-framework/vanilla-framework/pull/1290 This PR will include the new styles hoping this week 🤞